PelotonEppSdk/Models/EventResponse.cs

```diff
@@ -17,22 +17,24 @@ public class EventResponse : Response
         /// The name for the event.
         /// </summary>
         public string Name { get; set; }
+
+        public string FriendlyUrlPath { get; set; }
         /// <summary>
         /// The description for the event.
         /// </summary>
         public string Description { get; set; }
         /// <summary>
         /// The start datetime for the event.
         /// </summary>
-        public DateTime StartDatetime { get; set; }
```

Comment:
I'm fine with keeping the new name along with an increment to the major version. The new name is consistent with our coding style. The alternative is we revert the rename for now and when we have our next major release, then remember to go through an update the name. I'm in favor of doing it now to avoid building up technical debt.